src/Defaults.ts

```diff
@@ -0,0 +1,28 @@
+import * as ReactModal from 'react-modal';
+
+let _MODAL_ROOT: string = 'body';
+let _MODAL_TIMEOUT: number = 300;
+
+// tslint:disable-next-line:class-name
+/* istanbul ignore next */
+export abstract class defaults {
+    static get MODAL_ROOT(): string {
+        return _MODAL_ROOT;
+    }
+
+    static set MODAL_ROOT(root: string) {
+        _MODAL_ROOT = root;
+    }

```

Comment:
What about 
```typescript
export abstract class Defaults {
  static MODAL_ROOT = 'body';

  static MODAL_TIMEOUT = 300;

  static set APP_ELEMENT(appElement: string | HTMLElement) {
    ReactModal.setAppElement(appElement);
   }
}
```

Reply:
I think you'll be able to remove the tslint and istanbul disable

Reply:
The goal is that a user of the react-vapor lib will be able to set the modal root and modal timeout for all modals to whatever he wants. That's why I used getters and setters.

The modal root and the app element are 2 different things:
>The app element should not be a parent of the modal, to prevent modal content from being hidden to screenreaders while it is open.

https://reactcommunity.org/react-modal/#custom-parent